Doc 1617/add call to action for card out transactions (#587)
swan-io/docs: docs/topics/payments/cards/index.mdx

| `Released` | Card authorizations are released for specific reasons. Most of the time, the funds are captured when the merchant requests the actual debit. Authorizations might also be released by the merchant, and they can expire.<br /><br />When an authorization is released without a debit (clearing), the account's available balance increases by the amount of the authorization. |
| `Rejected` | Declined or refused card payments. For example, you, Swan, Mastercard rejected authorization for the payment, or the account's `Available` balance isn't sufficient to complete the card payment without resulting in a negative balance. |

> ## Rejected card transactions {#rejected}

Reviewer: Make this anchor rejected-card-transactions to make it clearer. The merchants' page also has a #rejected section.

> When a card payment is rejected, the account isn't charged. The **`reasonCode`** on the rejected transaction explains why.

Reviewer: ```suggestion
When a card payment is rejected, the account isn't charged. The **`reasonCode`** on the rejected transaction explains why. Share the following resolutions with your users to help them resolve common issues.
```
Add a framing sentence before the table to clarify the audience.

> When a card payment is rejected, the account isn't charged. The **`reasonCode`** on the rejected transaction explains why.

Reviewer: ```suggestion
When a card payment is rejected, the account isn't charged. The `reasonCode` in the rejected transaction's `statusInfo`explains why.
```

Highlights where they can find the reasonCode exactly.

Reviewer: Also doesn't need a bold around the fields since its already code-formatted.

> The following table lists the most common rejection codes for card transactions.

Reviewer: Can remove this since we're adding a "share this with users" in the paragraph above. Then remove the note callout later. 

```suggestion
Not all rejection codes are listed here — refer to the full [RejectedReasonCode enum](https://api-reference.swan.io/enums/rejected-reason-code/) for the complete list.
```

| `PinInvalid` | The PIN entered is incorrect. | Retry with the correct PIN. |
| `PinRequired` | The cardholder didn't enter their PIN. | Insert the card into the terminal and enter the PIN to complete the payment. |
| `PinRequiredForFurtherTransaction` | The cardholder must enter their PIN while completing a transaction to reactivate the card. | Complete a chip-and-PIN transaction to reactivate contactless functionality. |
| `SwanTechnicalErrorOccurred` | A technical error occurred. | Retry the transaction. If the problem persists, contact Swan Support with the transaction details. |
| `ThreeDsError` | There was an error with 3-D Secure (3DS). | Retry the transaction and complete the [3-D Secure authentication](#3ds). |
| `TransactionTypeNotAllowed` | This transaction type isn't allowed on the card. | Verify your [card settings](../../cards/index.mdx#settings). For example, you might have tried to withdraw money while withdrawals are disabled. |

:::note

> This table lists the most common rejection codes. View the complete [`RejectedReasonCode` enum](https://api-reference.swan.io/enums/rejected-reason-code/) in the API Reference for all possible values.

Reviewer: Can remove this and frontload it at the top.
